packages/api-codec/src/index.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017-2018 @polkadot/api-codec authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+// NOTE We are not re-exporting the index.ts from codec here. The reasoning being
+// that these should be enough to actually _use_ the codec, i.e. from a api-user
+// perspective these are the value classes. (Codec is for the cases where you need
+// to construct values dynamically)
+
+export { default as AccountId } from './AccountId';
+export { default as AccountIndex } from './AccountIndex';
+export { default as Address } from './Address';
+export { default as AuthorityId } from './AuthorityId';
+export { default as Balance } from './Balance';
+// NOTE Bft items are only used in internal structures
+// export * from './Bft;
+export { default as Block } from './Block';
+export { default as BlockNumber } from './BlockNumber';
+export { default as Bool } from './Bool';
+export { default as Bytes } from './Bytes';
+export { default as Extrinsic } from './Extrinsic';
+// NOTE These are currently only used internally, no direct mapping to Rust strings
+// export { default as H256 } from './H256';
+// export { default as H512 } from './H512';
+export { default as Hash } from './Hash';
+export { default as Header } from './Header';
+export { default as KeyValue } from './KeyValue';
+export { default as Metadata } from './Metadata';
+export { default as MisbehaviorReport } from './MisbehaviorReport';
+// NOTE Nonce is renamed to Index
+export { default as Index } from './Nonce';
+export { default as PropIndex } from './PropIndex';
+export { default as ProposalIndex } from './ProposalIndex';
+export { default as RawAddress } from './RawAddress';
+export { default as ReferendumIndex } from './ReferendumIndex';
+export { default as Signature } from './Signature';
+export { default as SignedBlock } from './SignedBlock';
+// NOTE Text is currently only used internally, export not needed
+// export { default as Text } from './Text';
```

Comment:
I need it in type-storage: for the well known keys, I manually type the documentation of those storage functions.

However I can also do `import Text from '@polkadot/api-codec/Text';`

Reply:
Yeap, I was thinking about that. Text is a funny one since it sounds cool, but apart from docs, actually never used - (apart from you on the API). So general API users should never need this one... Well, never say never...